superset/daos/dashboard.py

```diff
@@ -185,17 +185,15 @@ def update_charts_owners(model: Dashboard, commit: bool = True) -> Dashboard:
 
     @classmethod
     def delete(cls, items: Dashboard | list[Dashboard], commit: bool = True) -> None:
-        item_ids = [item.id for item in get_iterable(items)]
```

Comment:
See my previous comment.

Reply:
same here, there's some logic which is specific to dashboards.